core/src/main/java/org/apache/calcite/sql2rel/RelStructuredTypeFlattener.java

```diff
@@ -581,68 +555,52 @@ public void rewriteGeneric(RelNode rel) {
   }
 
   private void flattenProjections(RewriteRexShuttle shuttle,
-      List<? extends RexNode> exps,
-      List<String> fieldNames,
-      String prefix,
-      List<Pair<RexNode, String>> flattenedExps) {
+                                  List<? extends RexNode> exps,
```

Comment:
Please revert this change, Calcite uses such formatting.

Reply:
Fixed.